permissions/api_permissions/graphene_field_decorators.py

```diff
@@ -7,25 +6,20 @@
 
 def reservation_non_public_field(func: callable):
     def permission_check(*args, **kwargs):
-        if not settings.TMP_PERMISSIONS_DISABLED and not can_view_reservation(
-            args[1].context.user, args[0]
-        ):
-            return None
+        if can_view_reservation(args[1].context.user, args[0]):
```

Comment:
I know it is not really part of this PR, but would be really nice to give these args some names. I have no idea what they are 😄

Reply:
this is using the *args to comply the generic nature of this decorator, even though it is assuming there is more than one arg and the second clearly are expected to be `ResolveInfo` ... I'm not totally sure if we though should refactor this within this one 🤔 

This is used like this:
```
@reservation_non_public_field
def resolve_some_field(self, info):
    pass
```

I wouldn't refactor this as